02-esc/modal.js

```diff
@@ -101,22 +113,31 @@ function possibleNumberOfParticipants(min, max) {
 function openModalStepTwo(e) {
 	e.preventDefault();
 	modalDiv.innerHTML = `
+		<div class="modalContents">
         <div class="bookingStep2Content">
-        <h1>Book room "Title of room" (step 2)</h1>
+        <h1>Book room "${roomTitleforModal}" (step 2)</h1>
         <form action="">
-            <label for="name">Name</label>
+            <label class="modalInputLabels" for="name">Name</label>
+			<br>
```

Comment:
Line break elements (`<br>`) are rarely used nowadays, because most of the time that used to appear in olden times, they were used to achieve layout, which should not be done in HTML but with styling.

A better way of doing this would be to wrap each label/input pair in a div, or (as is common) wrap the input in the label, and make the label a `display: block` to put each on a new line.